package.json

```diff
@@ -52,6 +51,7 @@
     "moment-timezone": "0.5.37",
     "nyc": "15.1.0",
     "painterro": "1.2.78",
+    "playwright-core": "1.26.1",
```

Comment:
do we need this?

Reply:
which version of `npm` are you using? I removed it and did a clean + install and didn't see any warnings

Reply:
It happens post webpack compile. Note
`npm WARN @percy/playwright@1.0.4 requires a peer of playwright-core@>=1 but none is installed. You must install peer dependencies yourself.`
below:

```
$ npm run clean

> openmct@2.1.1-SNAPSHOT clean /Users/scott/workspace/openmct
> rm -rf ./dist ./node_modules ./package-lock.json

✔ ~/workspace/openmct [5031-replace-appjs-with-webpack-dev-server|✚ 1] 
19:08 $ npm install
npm WARN deprecated codecov@3.8.3: https://about.codecov.io/blog/codecov-uploader-deprecation-plan/

> core-js@3.25.3 postinstall /Users/scott/workspace/openmct/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> @percy/core@1.10.3 postinstall /Users/scott/workspace/openmct/node_modules/@percy/core
> node ./post-install


> openmct@2.1.1-SNAPSHOT prepare /Users/scott/workspace/openmct
> npm run build:prod


> openmct@2.1.1-SNAPSHOT build:prod /Users/scott/workspace/openmct
> webpack --config webpack.prod.js

webpack compiled successfully
npm WARN @percy/playwright@1.0.4 requires a peer of playwright-core@>=1 but none is installed. You must install peer dependencies yourself.
npm WARN ajv-keywords@3.5.2 requires a peer of ajv@^6.9.1 but none is installed. You must install peer dependencies yourself.

added 771 packages from 1200 contributors and audited 774 packages in 54.989s

86 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

Reply:
The `ajv` requirement is impossible to meet as far as I can tell.

Reply:
@scottbell this is installed separately in the `npx playwright@version install` step

Reply:
I only see this warning with node 14 and since that's the lower bound of what we support, I suppose let's keep this?

Reply:
Agreed

Reply:
@scottbell  I think we can resolve the `ajv` warning as well by adding

`"ajv": "6.9.1",`

Reply:
@scottbell do you mind removing the core pin? It'll be installed from other sources

Reply:
Ah I remember now:
```
$ npm run clean && npm install

> openmct@2.1.1-SNAPSHOT clean /Users/scott/workspace/openmct
> rm -rf ./dist ./node_modules ./package-lock.json

npm WARN deprecated codecov@3.8.3: https://about.codecov.io/blog/codecov-uploader-deprecation-plan/

> core-js@3.25.3 postinstall /Users/scott/workspace/openmct/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> @percy/core@1.10.3 postinstall /Users/scott/workspace/openmct/node_modules/@percy/core
> node ./post-install


> openmct@2.1.1-SNAPSHOT prepare /Users/scott/workspace/openmct
> npm run build:prod


> openmct@2.1.1-SNAPSHOT build:prod /Users/scott/workspace/openmct
> webpack --config webpack.prod.js

webpack compiled successfully
added 780 packages from 1200 contributors and audited 783 packages in 52.341s

86 packages are looking for funding
  run `npm fund` for details

found 1 moderate severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
```

it gives a "moderate severity vulnerability" if we add `ajv` `6.9.1`

Reply:
resolution is to keep the `ajv` warning because the required version produces an audit warning, and upgrading to resolve the audit warning breaks webpack.

Reply:
> @scottbell do you mind removing the playwrightcore pin? It'll be installed from other sources

If we do this, we get a warning from `npm` that we're not installing it:
```
$ npm run clean && npm install

> openmct@2.1.1-SNAPSHOT clean /Users/scott/workspace/openmct
> rm -rf ./dist ./node_modules ./package-lock.json

npm WARN deprecated codecov@3.8.3: https://about.codecov.io/blog/codecov-uploader-deprecation-plan/

> core-js@3.25.3 postinstall /Users/scott/workspace/openmct/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> @percy/core@1.10.3 postinstall /Users/scott/workspace/openmct/node_modules/@percy/core
> node ./post-install


> openmct@2.1.1-SNAPSHOT prepare /Users/scott/workspace/openmct
> npm run build:prod


> openmct@2.1.1-SNAPSHOT build:prod /Users/scott/workspace/openmct
> webpack --config webpack.prod.js

webpack compiled successfully
npm WARN @percy/playwright@1.0.4 requires a peer of playwright-core@>=1 but none is installed. You must install peer dependencies yourself.
npm WARN ajv-keywords@3.5.2 requires a peer of ajv@^6.9.1 but none is installed. You must install peer dependencies yourself.

added 771 packages from 1200 contributors and audited 774 packages in 55.911s

86 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```